apps/webpack.js

```diff
@@ -35,6 +35,11 @@ var baseConfig = {
           path.resolve(__dirname, 'static'),
         ],
         loader: "url-loader?limit=1024",
+      },
+      {
+        test: /\.jsx?$/,
+        loader: 'react-hot',
+        include: [path.resolve(__dirname, 'src')]
```

Comment:
need to make the use of this loader optional, so it doesn't get used for production builds.